CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ file is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and
 this project adheres to [Semantic
 Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+### Added
+
+- `confidence` field to `TiRule` structure as an optional f32 value
```

Comment:
Please modify this message like following:
- Added `confidence` field of type `Option<f32>` to `TiRule` structure